__tests__/app.test.js

```diff
@@ -202,8 +210,50 @@ describe("app", () => {
         .post("/api/articles/100/comments")
         .send(inputBody)
         .expect(404)
-        .then(({ body }) => {
-          expect(body.msg).toBe("article id not found");
+        .then(({ body: err }) => {
+          expect(err.msg).toBe("article id not found");
+        });
+    });
+  });
+  describe("PATCH /api/articles/:article_id", () => {
+    it("Request body accepts an object in the form { inc_votes: newVote }, updating votes, responds with the updated article", () => {
+      const inputBody = { inc_votes: 1 };
+      return request(app)
+        .patch("/api/articles/3")
+        .send(inputBody)
+        .expect(202)
```

Comment:
202 is for process accepted but not completed, whereas we're getting the updated article back, so 200 is more apt status code here